src/commonMain/kotlin/com/github/nanoflakes/PrimitiveNanoflake.kt

```diff
@@ -0,0 +1,11 @@
+package com.github.nanoflakes
+
+import kotlin.jvm.JvmInline
+
+/**
+ * A [Nanoflake](https://github.com/nanoflakes/nanoflakes).
+ *
+ * @param value the value of the nanoflake.
+ */
+@JvmInline
+value class PrimitiveNanoflake(override val value: Long) : Nanoflake
```

Comment:
I have a better name suggestion, `NanoflakeValue` or `SimpleNanoflake`

`PrimitiveNanoflake` is a bit weird.

Reply:
Simple sounds good.